seonshine_frontend/src/pages/forgotPassword/types.ts

```diff
@@ -0,0 +1,9 @@
+export type ForgotPasswordStepsType = 'enter_email' | 'verify_otp' | 'enter_new_password' | 'reset_successfully';
```

Comment:
We can use
`export enum ForgotPasswordStepEnum = {
ENTER_EMAIL = 1,
VERIFY_OTP = 2,
ENTER_NEW_PASSWORD = 3
...
}`

In this way, we can know step by step
But this is recommended. Just note, don't need to fix